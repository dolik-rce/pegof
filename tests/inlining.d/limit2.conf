input inlining.d/limit.peg
optimize inline,remove-group
inline-limit 0.224
