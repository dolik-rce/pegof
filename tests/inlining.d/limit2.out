main <- "A" X "B" X "C" X "D" X "E" X "F" X "G" X "H" X "I" X "J" X

# inline score for X is 1/sqrt(2 * 10) = 0.2236
X <- ";" "\n"*
